packages/wallet/ui/src/components/AppBar.jsx

```diff
@@ -62,6 +64,13 @@ const AppBar = () => {
         </a>
       </div>
       <div className={classes.appBarSection}>
+        <div className={classes.connector}>
+          {useChainBackend ? (
+            <ChainConnector></ChainConnector>
```

Comment:
I looked into code-splitting, but it seemed to be more complexity than it's worth for now. Using the async import would only save the app from loading a couple hundred lines of code.